about.html

```diff
@@ -0,0 +1,173 @@
+            BLOCKCHAIN ON-RAMP GLOBAL SUMMIT 2023
+          </h1>
+        </div>
+        <div id="hero-about-description">
+          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Error consequuntur enim hic ratione soluta mollitia ipsam aperiam tempora assumenda molestias consectetur nobis, velit facilis quam commodi illum obcaecati dolores nesciunt maiores ex blanditiis quia quas, voluptas culpa! Id, accusantium! Praesentium omnis molestiae in deserunt nisi blanditiis velit saepe placeat? Facilis.
+        </div>
+        <div id="contact-call">
+          <p>
+            Please contact us per Email for any further questions about Blockchin On-ramp Global Summit 2023!
+          </p>
+          <a href="mailto:njugunab655@gmail.com">njugunab655@gmail.com</a>
+        </div>
+      </div>
+      <div class="white">
+        <div id="about-logo">
+          <h2>Blockchain On-ramp Global Summit 2023 Logo</h2>
+          <hr class="section-heading-underline">
+          <p>The logo for Blockchain On-ramp Global Summit 2023 was decided during an art competition held from 14th February to 28th February.</p>
+          <div id="logo-img">
+            <!-- logo goes here -->
+            <img src="./images/blockchain-logo.png" alt="Blockchain summit logo">
+          </div>
+          <hr id="divider">
+        </div>
+        <div id="past-summits">
+          <h2>See the past Blockchain Summits</h2>
+          <hr class="section-heading-underline">
+          <p class="past-summits-p">Have a look at the last two Blockchain Global Summits which took place in Nairobi, Kenya and Addis Ababa, Ethiopia.</p>
+          <div id="summit-tiles">
+            <div class="summit latest">
+              <h3>2021</h3>
+              <p class="past-summits-p">Blockchain On-ramp Global Summit 2021 in Nairobi, Kenya</p>
+            </div>
+            <div class="summit second-latest">
+              <h3>2020</h3>
+              <p class="past-summits-p">Blockchain On-ramp Global Summit 2020 in Addis Ababa, Ethiopia</p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="partners">
+        <h2>Partner</h2>
+        <hr class="section-heading-underline">
+        <!-- partner logos go here -->
+        <div id="partner-logos">
+          <img src="./images/partners/chainlink-link-logo.svg" alt="Chainlink">
+          <img src="./images/partners/cardano-ada-logo.svg" alt="Cardano">
+          <img src="./images/partners/solana-sol-logo.svg" alt="Solana">
+          <img src="./images/partners/Ethereum-Landscape-Black-Logo.wine.svg" alt="Ethereum">
+          <img src="./images/partners/google.svg" alt="Google">
+        </div>
+      </div>
+    </section>
+  </main>
+  <footer>
+    <img src="./images/blockchain-logo.png" alt="Logo" class="footer-logo">
+    <div class="footer-description">
+      <p>2023 Blockchain On-ramp Summit.</p>
+      <p>Some rights reserved.</p>
+    </div>
+  </footer>
```

Comment:
- [ ] While you do a great effort 😃 I see that the padding of the footer is less than the Behance design, so please 😊 try to increase the padding vertically to be like the Behance 💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/225305686-8690afc3-4725-4df8-8c20-e6ce0cc2e561.png)  |    ![image](https://user-images.githubusercontent.com/83360636/225305623-dcb6993c-de17-4e7b-a739-29873b898a6e.png)  |